lib/galaxy/datatypes/sequence.py

```diff
@@ -1206,25 +1206,24 @@ def sniff_prefix(self, file_prefix: FilePrefix) -> bool:
         >>> Axt().sniff( fname )
         False
         """
-        headers = get_headers(file_prefix, None)
-        if len(headers) < 4:
+        headers = get_headers(file_prefix, None, count=4, comment_designator="#")
+        if not (
+            len(headers) == 4
+            and len(headers[0]) == 9
+            and headers[0][0].isdigit()
```

Comment:
According to the standard:

```suggestion
            and headers[0][0] == "0"
```